databrowser/src/layouts/menu/MenuPopover.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <Popover class="relative">
+    <PopoverButton class="flex space-x-4">
+      <IconMenu />
+      <span>Menu</span>
+    </PopoverButton>
+    <PopoverPanel
+      class="inline-flex absolute z-10 py-8 px-2 bg-white divide-x shadow"
+    >
+      <DatasetMenu />
+    </PopoverPanel>
```

Comment:
On mobile the panel is full screen and only one section is always shown.

<img width="621" alt="Bildschirmfoto 2022-01-12 um 13 19 41" src="https://user-images.githubusercontent.com/3337260/149139141-5fb4daa9-4d23-4ddf-80fd-00c49cf6c0b9.png">

Reply:
On desktop the panel is also full screen and the different sections are shown side by side.

<img width="1267" alt="Bildschirmfoto 2022-01-12 um 13 21 08" src="https://user-images.githubusercontent.com/3337260/149139365-d619b577-bac0-42ed-aeba-2ceb639ea5ba.png">